helm/flink-operator/values.yaml

```diff
@@ -58,7 +58,7 @@ webhook:
 operatorConfiguration:
   create: true
   # Set append to false to replace configuration files
-  # append: true
+  append: true
```

Comment:
So now the default behavior is append, not replace. Right?

Reply:
Before this change, append was the default even if not specified. That was the source of the helm template complications :D 

Now it is explicitly set to append=true in the default values.yaml  but if you omit it , it will be false.

Reply:
I think this is more predictable behaviour now

Reply:
Make sense.